scripts/create_AR_DeployInst.py

```diff
@@ -222,11 +227,149 @@ def determine_deployment_order(reactor_prototypes):
     return reactor_order
 
 
+def update_di(di, prototype, num_rxs, build_time, lifetime):
+    '''
+    Update a dictionary of a DeployInst
+
+    Parameters:
+    -----------
+    di: dictionary
+        dictionary defining the DeployInst, the top-level key
+        is 'DeployInst', the next level keys are 'prototypes',
+        'n_build', 'lifetimes', and 'build_times'. Each of the
+        second level keys have a nested dictionary of
+        'val':list
```

Comment:
The format of the dictionary is: `{'DeployInt':{'prototypes':{'val':[]}, 'n_build':{'val':[]}, 'buildtimes':{'val':[]}, 'lifetimes':{'val':[]}}` and this structure is because of the xml structure for the DeployInst:
`<DeployInst>
  <prototypes>
    <val>str</val>
    <val>str</val>
    ....
  </prototypes>
  ...
</DeployInst
`

Reply:
Ok! I was a little confused there, but that makes sense